scripts/globals/mobskills.lua

```diff
@@ -375,6 +383,15 @@ xi.mobskills.mobAddBonuses = function(caster, target, dmg, ele) -- used for SMN
     dmg             = math.floor(dmg * dayWeatherBonus)
 
     local burst = calculateMobMagicBurst(caster, ele, target)
+    if
+        burst > 1.0 and
+        caster:getPetID() >= 0 -- all pets except charmed pets can get magic burst message, but only with petskill action
```

Comment:
`>=`?

Reply:
Multiplier is default 1

Oh you mean pet id, whoops

Reply:
Fixed, nice catch